application-services/custom/secrets/README.md

```diff
@@ -2,119 +2,85 @@
+       - vault
+       environment:
+         ADD_SECRETSTORE_TOKENS: 'app-secrets'
    ```
 
-5. Run EdgeX security services in docker to generate a Vault token for our app service:
-
-   1. Run the secret store services in EdgeX using:
-
-      `docker-compose -f <docker-compose file name> up -d vault-worker`
-
-      Verify that the following services are running:
-         - edgex-secrets-setup
-         - edgex-vault
-         - edgex-vault-worker
-   2. Wait for about a minute to ensure tokens are created for the services that were specified in the docker compose section of vault-worker service for `ADD_SECRETSTORE_TOKENS` environment variable. Verify that the token `secrets-example` exist under `/tmp/edgex/secrets`.
-
-5. Run the secrets example service:  
-      ```
-      docker-compose -f <docker-compose file name> up -d app-service-secrets-example
-      ```
-
-      Check the app's logs to make sure no errors occurred during startup:
-   
-      ```
-      docker logs app-service-secrets-example
-      ```
-   
-6. Follow the instructions in [Storing and Getting Secrets](#storing-and-getting-secrets) in order to test storing and retrieving secrets from the secret store.
-
-## Run the app natively (for dev/debug)
+   This creates a Vault back SecretStore for our example and populates it with then known `redisdb` secret. In addition creating the SecretStore, service's can request known secrets and to be added to the API Gateway. See the [Configuring Add-on Service](https://docs.edgexfoundry.org/2.0/security/Ch-Configuring-Add-On-Services/) security documentation for complete details.
 
-The default configuration settings are used for running natively. The application service is already configured for running Vault in dev-mode (with http).  We need to run Vault, enable its secrets engine, then run our service. 
+4. Run the Edgex services in Docker using the modified compose file from above.
 
-1. Run vault using the docker compose file in this directory. 
+   Run the following command from the same folder the compose file resides.
 
-   `docker-compose -f docker-compose.yml up -d`
-
-2. You will need to manually set the current Vault token in configuration.
-
-   - Find the root token for Vault in the logs of the container:
-
-     `docker logs <container-id>`
-
-   ![image-20200224130525112](./root-token.png)
+   ```
+   docker-compose -p edgex up -d
+   ```
 
-   - Copy the token to the *'root_token'* field in the token file (./res/*token.json*). Verify that `SecretStore.TokenFile` and  `SecretStoreExclusive.TokenFile` are already configured with ./res/*token.json* as the token file path.
+   Now all the EdgeX service will be running. This can be verified by running the following command:
 
-3. Use docker exec to run commands on the running vault container instance.
+   ```
+   docker-compose -p edgex ps
+   ```
 
-   `docker exec -it <container_id> /bin/sh`
+   Which will output the following:
+
+   ```bash
```

Comment:
use console probably looks better
```suggestion
   ```console
```

Reply:
Oh, didn't know that was an option... will try it.

Reply:
console isn't an option.

Reply:
adda